shared/clinical-code-sets/conditions/allergy/1/README.md

```diff
@@ -0,0 +1,13 @@
+# Allergy codes
+
+Developed from https://getset.ga.
```

Comment:
Can you add a line explaining what the code set covers e.g. Any code that indicates a diagnosis of a new allergy.